### DISCUSSION TITLE
Inline code in the document editor

### DISCUSSION THREAD

**Discussion by ScottAgirs** [keystonejs/keystone#7387] [Questions] [answered]
Hey folks, sorry to ping here instead of Discussions (for features), but I really feel that this is more of an issue than a feature request.

Inline code for rich-text editor is quite important, dare I say - critical.
Currently, unless I'm missing something it appears that it's completely absent from the scheme - am I wrong? (i hope so).

And if I'm not wrong - could we hope for it to be added in near future?

Thanks so much for the awesome work and for considering fellas 🙌🏼

**Comment by dcousens** [comment, marked as answer]
Hi @ScottAgirs,

What feature did you feel was missing for the document field in our https://keystonejs.com/docs/guides/document-field-demo?

You can type ```` ``` ```` to use code blocks.

**edit:** You can type ``` ` ``` around your text for inline code formatting.

**Reply by ScottAgirs** [reply]
Hey @dcousens thanks for the reply and moving the issue to here.

That's true, the code blocks are awesome and work like a charm, but I feel like the `inline-code` leafs are missing, no?

**Reply by dcousens** [reply]
@ScottAgirs you can produce inline code blocks using the typical markdown syntax of `` ` `` around your text.

**Reply by ScottAgirs** [reply]
Ohh gosh, my apologies 🤦🏼‍♂️
